### ISSUE TITLE
هشدار 

### ISSUE THREAD

**Issue by 1008999762** [bitpay/bitcore#3725]
Hello. I sent you a message with account ID 1008999762. Unfortunately, you did not reply.
I have waited until now to receive an answer.
I identified about 42 people around me who are working with your Bitcore collection.
So far, I have not said anything about what happened to my capital.
I also ask you to transfer my money and capital to my Tron address according to my payment invoice on April 13, 2024, when I deposited Tether for you and I did not receive any receipt.
If you deposit, you will be fine and we will be fine.
If you do not deposit, in the first step, I will introduce these 42 people to your system, and they will also inform the others who work with your collection, and together we will go to the FATA police and relevant bodies and file a fraud complaint against you. do.
We are advertising your tricks and scams on all the big corners of Instagram, Telegram, Facebook, etc., so that your work and business will be destroyed, and you will no longer be able to steal people's capital like this.
We are doing our best to destroy this system and create a No to Bitcore scam campaign.
We will complain about your fraud on Telegram, BitPay, GitHub, etc. and we will do the necessary follow-up.
Therefore, it is better to pay this invoice, which has been paid and not settled, to the address of my Tron wallet before taking the mentioned actions.
TJAkp2jy5nrF2MS36YkEkDuCp1iCZsbVJ3

And bless you and keep us safe, and then do whatever you like as before.
The choice is yours.

I created a new invoice on April 13, 2024.
Account ID: 1008999762
I transferred the amount of 3033 Tether to the following address.

TGRee3p5TnVXuhYVE5f7BkC9AvNWLyGSSu
Network type TRC20
┈┅┅━━✦━━┅┅┈
Selected miner details:

Avalon

☑️ Contract duration: 3 days
☑️ Daily extraction: 1516.5 dollars
☑️ Total extraction: $4549.5
☑️ Daily-automatic payment
hello

**Comment by 1008999762** [comment]
Hello. I sent you a message with account ID 1008999762. Unfortunately, you did not reply.
I have waited until now to receive an answer.
I identified about 42 people around me who are working with your Bitcore collection.
So far, I have not said anything about what happened to my capital.
I also ask you to transfer my money and capital to my Tron address according to my payment invoice on April 13, 2024, when I deposited Tether for you and I did not receive any receipt.
If you deposit, you will be fine and we will be fine.
If you do not deposit, in the first step, I will introduce these 42 people to your system, and they will also inform the others who work with your collection, and together we will go to the FATA police and relevant bodies and file a fraud complaint against you. do.
We are advertising your tricks and scams on all the big corners of Instagram, Telegram, Facebook, etc., so that your work and business will be destroyed, and you will no longer be able to steal people's capital like this.
We are doing our best to destroy this system and create a No to Bitcore scam campaign.
We will complain about your fraud on Telegram, BitPay, GitHub, etc. and we will do the necessary follow-up.
Therefore, it is better to pay this invoice, which has been paid and not settled, to the address of my Tron wallet before taking the mentioned actions.
TJAkp2jy5nrF2MS36YkEkDuCp1iCZsbVJ3

And bless you and keep us safe, and then do whatever you like as before.
The choice is yours.

I created a new invoice on April 13, 2024.
Account ID: 1008999762
I transferred the amount of 3033 Tether to the following address.

TGRee3p5TnVXuhYVE5f7BkC9AvNWLyGSSu
Network type TRC20
┈┅┅━━✦━━┅┅┈
Selected miner details:

Avalon

☑️ Contract duration: 3 days
☑️ Daily extraction: 1516.5 dollars
☑️ Total extraction: $4549.5
☑️ Daily-automatic payment
hello

**Comment by 1008999762** [comment]
Hello. I sent you a message with account ID 1008999762. Unfortunately, you
did not reply.
I have waited until now to receive an answer.
I identified about 42 people around me who are working with your Bitcore
collection.
So far, I have not said anything about what happened to my capital.
I also ask you to transfer my money and capital to my Tron address
according to my payment invoice on April 13, 2024, when I deposited Tether
for you and I did not receive any receipt.
If you deposit, you will be fine and we will be fine.
If you do not deposit, in the first step, I will introduce these 42 people
to your system, and they will also inform the others who work with your
collection, and together we will go to the FATA police and relevant bodies
and file a fraud complaint against you. do.
We are advertising your tricks and scams on all the big corners of
Instagram, Telegram, Facebook, etc., so that your work and business will be
destroyed, and you will no longer be able to steal people's capital like
this.
We are doing our best to destroy this system and create a No to Bitcore
scam campaign.
We will complain about your fraud on Telegram, BitPay, GitHub, etc. and we
will do the necessary follow-up.
Therefore, it is better to pay this invoice, which has been paid and not
settled, to the address of my Tron wallet before taking the mentioned
actions.
TJAkp2jy5nrF2MS36YkEkDuCp1iCZsbVJ3

And bless you and keep us safe, and then do whatever you like as before.
The choice is yours.

I created a new invoice on April 13, 2024.
Account ID: 1008999762
I transferred the amount of 3033 Tether to the following address.

TGRee3p5TnVXuhYVE5f7BkC9AvNWLyGSSu
Network type TRC20
┈┅┅━━✦━━┅┅┈
Selected miner details:

Avalon

☑️ Contract duration: 3 days
☑️ Daily extraction: 1516.5 dollars
☑️ Total extraction: $4549.5
☑️ Daily-automatic payment
hello

در تاریخ دوشنبه ۲۲ آوریل ۲۰۲۴،‏ ۲۳:۰۴ kajoseph ***@***.***>
نوشت:
